exiv2node.cc

```diff
@@ -5,6 +5,7 @@
 #include <unistd.h>
 #include <string>
 #include <map>
+#include <list>
```

Comment:
For most cases this list will contain single element. And all manipulations, such as conversion to v8::Array, happen to iterate through all values. Given that we don't need random access to elements and want the quick addition, "list" should be a good choice here.